source/index.htm.erb

```diff
@@ -82,7 +108,7 @@ description: "Gridcoin is a cryptocurrency which rewards volunteer distributed c
                         </tr>
                         <tr>
                             <td>
-                                Earn 10 GRC per block staked
+                                Constant Block Rewards

```

Comment:
Might need to be phrased differently.  I think the old version leads to less confusion, but am open to a different format.

Reply:
I could add a little hover notation to it so users can hover over for a brief explanation. To help reduce confusion.

Reply:
More specifically, I can either use the <abbr> tag or a more fancy approach, use bootstrap tooltips